code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/TokenUtils.java

```diff
@@ -104,6 +95,25 @@ public Account getAccountFromJWT() {
         return jsonData;
     }
 
+    private String removeBearer(String jwtKey) {
+        String toParse;
+        if (StringUtils.isBlank(jwtKey)){
+            return jwtKey;
+        }
+        String[] tokenArr = jwtKey.split("\\s+");
+        if (tokenArr.length == 2) {
+            if (!StringUtils.equalsIgnoreCase("bearer", StringUtils.trim(tokenArr[0]))) {
+                return null;
```

Comment:
if it doesn't start with bearer and has a space it is not a jwt or a id for one so we want to return null. We don't want an exception because we throw those in a different place.